delfin/common/config.py

```diff
@@ -119,6 +119,24 @@
                .DEF_PERFORMANCE_HISTORY_ON_RESCHEDULE,
                help='default history(in sec) to be collected on a job '
                     'reschedule'),
+    cfg.BoolOpt('enable_dynamic_subprocess',
+                default=False,
+                help='Enable dynamic subprocess metrics collection'),
+    cfg.IntOpt('process_cleanup_interval',
+               default=60,
+               help='Background process cleanup call interval in sec'),
+    cfg.IntOpt('task_cleanup_delay',
+               default=10,
+               help='Delay for task cleanup before killing child in sec'),
+    cfg.IntOpt('group_change_detect_interval',
+               default=30,
+               help='Local executor group change detect interval in sec'),
+    cfg.IntOpt('max_storages_in_child',
+               default=5,
+               help='Max storages handled by one local executor process'),
+    cfg.IntOpt('max_childs_in_node',
```

Comment:
Is this default value ok? It means allowing 100000 process?

Reply:
Currently, we do not restrict number of processes created. Used a large number as default, before raising exception. Also, delete of process, when having no storages to handle takes about 90 seconds. So large number will provide a buffer, if we create and delete storage frequently

Reply:
@ThisIsClark This can be customised based on user enthronement based on their deployment  configuration. For example this can be set to number of cores available in a node.

Reply:
We are not restricting storages, we allow storages to get registered.